docs/directives/needuml.rst

```diff
@@ -152,6 +152,44 @@ and can't be ``diagram``.
       B -> C: Hi
       C -> B: Hi there
 
+save
+~~~~
+
+Specifies the file path to store generated Plantuml-code of current ``needuml``. This given file path can be relative path
+or file name, e.g. ``needuml_group_A/my_needuml.puml`` or ``my_needuml.puml``.
+
+The file will be created and written to relative to current builder output directory during each build by 
+using builder :ref:`needumls_builder` or other builder like `html` with configuration option :ref:`needs_build_needumls` configured.
+
+If given file path already exists, it will be overwritten.
+
+|ex|
+
+.. code-block:: rst
+
+   .. int:: Test needuml save
+      :id: INT_001
+
+      .. needuml::
+         :save: _build/_out/my_needuml.puml
+
+         Alice -> Bob: Hi Bob
+         Bob --> Alice: Hi Alice
+
+In this example, if builder :ref:`needumls_builder` is used, the plantuml-code will be exported to file at `outdir` of current builder,
+e.g. `_build/needumls/_build/_out/my_needuml.puml`.
```

Comment:
Not so nice examples having twice `_build` here and `_out.
Use the same example folder names as in the other descritpions.